backend/app/models/spree/backend_configuration.rb

```diff
@@ -46,7 +46,8 @@ def initialize(
         label: nil,
         partial: nil,
         url: nil,
-        position: nil
+        position: nil,
+        match_path: nil
```

Comment:
Would you mind expanding the above comment to explain what `match_path` is used for?

Reply:
Could I help to add the comment?

Reply:
@andyyang We would love to have you help out! I think Thomas has this covered. Would you take a look at https://github.com/solidusio/solidus/issues for another issue? Thanks!